docs/httplug.md

```diff
@@ -5,22 +5,39 @@ Httplug is an abstraction for HTTP clients. There are two main use cases:
 1. Usage in a project
 2. Usage in a reusable package
 
-In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface` or throws an exception that implements `Http\Client\Exception`.
+In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface`
+or throws an exception that implements `Http\Client\Exception`.
+
+There is also the HttpAsyncClient, available in [php-http/httplug-async](https://packagist.org/packages/php-http/httplug-async), which provides the `sendAsyncRequest` method to send a request asynchronously and returns a `Http\Client\Promise`.
+It can be used later to retrieve a PSR-7 `ResponseInterface` or an exception that implements `Http\Client\Exception`.
 
-See the [tutorial](tutorial.md) for a concrete example.
 
+<p class="text-warning">
+    Contract for the HttpAsyncClient is experimental until [PSR about Promise is released](https://groups.google.com/forum/?fromgroups#!topic/php-fig/wzQWpLvNSjs).
+    Once it is out, we will use this interface in the main client and deprecate the separated repository.
+</p>
+
+See the [tutorial](tutorial.md) for a concrete example.
 
 ## Httplug implementations
 
-Httplug implementations typically are either HTTP clients of their own, or they are adapters wrapping existing clients like Guzzle 6. In the latter case, they will depend on the required client implementation, so you only need to require the adapter and not the actual client.
+Httplug implementations typically are either HTTP clients of their own, or they are adapters wrapping existing clients like Guzzle 6.
```

Comment:
i had it like this to make it more clear that implementations can be one or the other things. reading over it quickly could otherwise lead to confusion as we talk about clients and then stuff that wraps clients (the later are a different kind of clients)